src/frontend/packages/core/src/shared/components/list/list-types/app-autoscaler-metric-chart/app-autoscaler-metric-chart-card/combo-chart/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './combo-chart.component';
```

Comment:
Think this should be a module that exports these two components and imported as a module where needed